src/common/components/FieldDatepicker/FieldDatepicker.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
`LoadingPane` is out of `components` folder, should we have additional folder for this one?

Reply:
or `LoadingPage` should be moved under `components`

Reply:
OK